tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/settings/JmapSettings.scala

```diff
@@ -40,6 +41,8 @@ object JmapSettings {
   case class JmapSettingsValue(value: String) extends AnyVal
 }
 
+trait JmapSettingEntry
```

Comment:
can we have another name?
it is bit confusing with `JmapSettingsEntry`  https://github.com/linagora/tmail-backend/blob/a39701c757e7c6aeac357a20ef196e5d18912876/tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/model/JmapSettingsGet.scala#L30C12-L30C29

Reply:
Or refactoring? Actually after thinking again, I think JmapSettingsEntry here would be more accurate (it's one entry of the settings) than yours...

For yours -> JmapSettings? or JmapSettingsObject? or JmapSettingsSingleton? 

The term `Entry` makes me think of something small, atomic

Reply:
I'm fine with both